File: leetcode/344_Reverse_String.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        for item in range(len(s)//2):
            temp = s[item]
            s[item] = s[len(s) - (1 + item)]
            s[len(s) - (1 + item)] = temp
            #print(s)

File: leetcode/344.Reverse_String.cs
//https://leetcode.com/problems/reverse-string/
public class Solution {
    public void ReverseString(char[] s) {
        helper(s,0,s.Length -1);
    }
     public void helper(char[] s, int left, int right)
        {
            if(left >= right) return;
            char temp = s[left];
            s[left++] = s[right];
            s[right--] = temp;
            helper(s, left, right);
        }
}
